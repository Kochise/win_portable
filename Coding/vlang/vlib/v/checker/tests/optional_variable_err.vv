fn main() {
	_ := ?bool(false)
}
