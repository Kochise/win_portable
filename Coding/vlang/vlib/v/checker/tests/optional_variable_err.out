vlib/v/checker/tests/optional_variable_err.vv:2:7: error: casting to optional type is forbidden
    1 | fn main() {
    2 |     _ := ?bool(false)
      |          ~~~~~~~~~~~~
    3 | }
